### PR TITLE
Invent new highlight indicator for "prefix" line

### PR DESCRIPTION
**Pull request by a1exsh**
This is ugly, but it works.

### CODE REVIEW

File: pg_view.py [hunk 1]
```diff
@@ -2590,13 +2590,20 @@ def display_prefix(self, collector, header):
         if prefix:
             prefix_len = len(prefix)
             prefix_newline = prefix[-1] == '\n'
+            if prefix_len >= 2:
```
Comment (alexeyklyukin) [inline pg_view.py]: I'm not sure that magical double newline is better than magical "(offline)" at the end of the string.

Comment (a1exsh) [inline pg_view.py]: Well, I think it is.  You can apply it to other messages which do not end with "offline".  Both approaches are ugly IMO, as I've said already.

Comment (alexeyklyukin) [inline pg_view.py]: But the other messages will display some other status (i.e. restarting, reloading, whatever), that can be used the same way as offline. I think using them is better than relying on some "invisible characters". For instance, here you also missing changes in the prefix length truncation code.

Comment (a1exsh) [inline pg_view.py]: Yeah, my overall feeling is that this is too fragile to touch.

**Comment by LappleApple** [comment]
@a1exsh This remains unmerged; do you want a Zappr thumbsup to go forward?

**Comment by LappleApple** [comment]
@a1exsh @alexeyklyukin Pinging again ...

**Comment by a1exsh** [comment]
I'd close this, it's such an ugly hack.

**Pull request closed without merging**